### PR TITLE
feat: write circle geometry as polygon

### PR DESCRIPTION
**Pull request by oterral**
# How to

Transform Circle geometry into Polygon in KML

# Others

- [x] It's not a hack or at least an unauthorized hack :).
- [x] The images added are optimized.
- [x] Everything in ticket description has been fixed.
- [x] The author of the MR has made its own review before assigning the reviewer.
- [x] The title means something for a human being and follows the [conventional commits](https://www.conventionalcommits.org/) specification.
- [x] The title contains [WIP] if it's necessary.
- [x] Labels applied. if it's a release? a hotfix?
- [x] Tests added.

### CODE REVIEW

File: src/components/FeatureExportButton/README.md [hunk 1]
```diff
@@ -20,22 +21,40 @@ import FeatureExportButton from 'react-spatial/components/FeatureExportButton';
...
+          color: '#3399CC',
+          width: 4,
+        }),
+        fill: new Fill({
+          color: '#3399CC',
+          width: 4,
+        }),
+      }),
+      // image: new Icon({
+      //   anchor: [0.5, 46],
```
Comment (danji90) [inline src/components/FeatureExportButton/README.md:35]: Maybe clean these. Don't see the point of keeping them

File: src/components/FeatureExportButton/README.md [hunk 2]
```diff
@@ -20,22 +21,40 @@ import FeatureExportButton from 'react-spatial/components/FeatureExportButton';
...
         new Feature({
-          geometry: new Point([819103.972418, 6120013.078324]),
-        }),
-        new Feature({
-          geometry: new Point([873838.856313, 6106009.575876]),
+          geometry: new Circle([819103.972418, 6120013.078324],10000),
         }),
+        // new Feature({
+        //   geometry: new Point([819103.972418, 6120013.078324]),
+        // }),
```
Comment (danji90) [inline src/components/FeatureExportButton/README.md:57]: same

**Review by danji90 — COMMENTED** [review]
Looks good, just some comment cleaning.
Is there a branch where I can test his?

File: src/components/FeatureExportButton/README.md (hunk 1 above)
Comment (oterral) [reply src/components/FeatureExportButton/README.md]: I put back the original example

File: src/components/FeatureExportButton/README.md (hunk 2 above)
Comment (oterral) [reply src/components/FeatureExportButton/README.md]: I put back the original example

**Pull request merged**